yt_idefix/data_structures.py

```diff
@@ -370,6 +370,9 @@ def _get_meta_file(
         ):
             return str(file.absolute())
         else:
+            ytLogger.warning(
```

Comment:
I'd rather issue an actual warning instead of a logger warning.

Reply:
Could you explain what's the difference between them and how to choose them?

Reply:
Thanks, I got it!